BlueprintUI/Sources/Focus/FocusBinding.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+
+/// A two-way binding between a focusable element's backing view and a `FocusState`-wrapped property.
+///
+/// You can use bindings to add focus support to custom elements.
+///
+/// TODO
+///
+public struct FocusBinding {
+    public typealias Callback = () -> Void
+
+    public let trigger = FocusTrigger()
+    public var onFocus: Callback?
+    public var onBlur: Callback?
```

Comment:
Is `blur` a common way to describe unfocusing from the perspective of interactivity?

I get the analogy, but it took me a minute to put this together. I'm wondering if `onUnfocus` would be more clear, but if others are comfortable with using `blur` in this context then I'm a fan!

Reply:
Might be good to align w/ the terminology from UIKit or SwiftUI; which for UIKit is resign and become first responder? (Unsure what language SwiftUI uses)

Reply:
I think the SwiftUI analogous way would be an onFocusChange with a bool isFocused - though SwiftUI's focus state is really quite different.

I agree onBlur was not initially obvious, but if it's well documented I think it's okay.

Reply:
I've matched the SwiftUI API where possible, but this part is plumbing that isn't exposed in SwiftUI.

> Is blur a common way to describe unfocusing from the perspective of interactivity?

Yes! Perhaps most notably in [the HTML DOM](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/blur), the world's most popular declarative UI framework. ;)

Reply:
Do we think this is a term iOS engineers will understand? After all, iOS engineers usually don't write HTML.

Reply:
I might be setting myself up for disappointment, but I'm pretty confident that folks will be able to figure this out!